spec/controllers/criteria_controller_spec.rb

```diff
@@ -199,23 +199,49 @@
       end
 
       describe '#new' do
-        before do
-          get_as instructor,
-                 :new,
-                 params: { course_id: course.id, assignment_id: assignment.id },
-                 format: :js
-        end
+        context 'assignment marks are not released' do
```

Comment:
For `context` blocks, the description should generally start with the word "when"